tests_integration/__tests__/file-info.js

```diff
@@ -106,16 +106,17 @@ describe("extracts file-info with inferredParser=foo when a plugin is hand-picke
 });
 
 test("API getFileInfo with no args", () => {
-  // TODO: change this to `rejects.toThrow()` when we upgrade to Jest >= 22
-  // https://github.com/facebook/jest/issues/3601
-  expect.assertions(1);
-  return prettier.getFileInfo().catch(err => {
-    expect(err).toBeDefined();
-  });
+  expect(prettier.getFileInfo()).rejects.toThrow(
+    // this error info might change depends on how getFileInfo works
+    new TypeError(`Cannot read property 'replace' of undefined`)
```

Comment:
Where does this error message come from?

Reply:
see comment above.

for now its come from undefined.replace

too strict?

Reply:
or maybe we should throw a error with specific info?